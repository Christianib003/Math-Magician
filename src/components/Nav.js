import React from 'react';
import { NavLink } from 'react-router-dom';

export default class Nav extends React.PureComponent {
  render() {
    return (
      <nav className="navBar">
        <NavLink className="header" to="/">
          <h3 className="title">Math Magicians</h3>
        </NavLink>
        <div className="navItems">
          <ul className="navMenu">
            <li><NavLink className="homeLink" to="/">Home </NavLink></li>
            <li><NavLink className="calculatorLink" to="/calculator">Calculator</NavLink></li>
            <li><NavLink className="quotesLink" to="/quotes">Quotes</NavLink></li>
          </ul>
        </div>
      </nav>
    );
  }
}
