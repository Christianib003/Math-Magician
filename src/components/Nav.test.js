import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Nav from './Nav';

it('renders correctly', () => {
  const nav = renderer
    .create(
      <Router>
        <Nav />
      </Router>,
    )
    .toJSON();
  expect(nav).toMatchSnapshot();
});
