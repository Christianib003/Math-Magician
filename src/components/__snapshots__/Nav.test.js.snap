// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navBar"
>
  <a
    aria-current="page"
    className="header active"
    href="/"
    onClick={[Function]}
  >
    <h3
      className="title"
    >
      Math Magicians
    </h3>
  </a>
  <div
    className="navItems"
  >
    <ul
      className="navMenu"
    >
      <li>
        <a
          aria-current="page"
          className="homeLink active"
          href="/"
          onClick={[Function]}
        >
          Home 
        </a>
      </li>
      <li>
        <a
          className="calculatorLink"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className="quotesLink"
          href="/quotes"
          onClick={[Function]}
        >
          Quotes
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
